packages/zed-wasm/test/browser/parse.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { test, expect } from '@playwright/test';
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('/test/pages/parse.html');
+  await page.getByText('Wasm Ready').waitFor({ state: 'attached' });
+});
+
+test('parse a simple zed string', async ({ page }) => {
+  await page.fill('input', 'zed := "world"');
+  await page.click('button');
+  await expect(page.locator('pre')).toContainText('"kind": "OpAssignment"');
```

Comment:
On the zed side of things we avoid not having tests that introspect the contents of the ast since this is often subject to change. I think I'd just have a test here that looked for a parser error and verifies the error message- something like `put x :=`.